frontend/src/app/components/containers/complaints/outcomes/hwcr-complaint-assessment.tsx

```diff
@@ -251,6 +251,7 @@ export const HWCRComplaintAssessment: FC = () => {
 
       dispatch(upsertAssessment(id, updatedAssessmentData));
       setEditable(false);
+      dispatch(setIsInEdit({ showSectionErrors: false }));
```

Comment:
hi @dmitri-korin-bcps sorry, this will cause this case: 
1. Open Edit mode in other sections: Assessment, Prevention, Equipment, etc..
2. Try to close the complaint -> error shows in Assessment, Prevention, Equipment as expected
3. Click Save in Assessment
4. Errors disappeared in other sections (which we not want) 

So just remove this line and you might want to remove `assessmentRelatedError` in `hwcr-prevention-education.tsx` (and that should be it 🙂) Sorry I should have notified you earlier